test/helpers/cilium.go

```diff
@@ -456,8 +457,20 @@ func (s *SSHMeta) PolicyImportAndWait(path string, timeout time.Duration) (int,
 	}
 	s.logger.WithFields(logrus.Fields{
 		logfields.Path:           path,
-		logfields.PolicyRevision: revision}).Infof("before importing policy")
-	res := s.ExecCilium(fmt.Sprintf("policy import %s", path))
+		logfields.PolicyRevision: revision}).Info("before importing policy")
+
+	s.logger.WithFields(logrus.Fields{
+		logfields.Path: path}).Info("validating policy before importing")
+
+	res := s.ExecCilium(fmt.Sprintf("policy validate %s", path))
```

Comment:
I didn't know about validate command, this is great :-)

Reply:
:D